content/docs/5.1/concepts/explore-tikv-features/replication-and-rebalancing.md

```diff
@@ -0,0 +1,144 @@
+```
+
+The command above will output the following content:
+```txt
+...
+Starting component `playground`: /home/pingcap/.tiup/components/playground/v1.5.0/tiup-playground --mode tikv-slim --kv 3
+Using the version v5.0.2 for version constraint "".
+
+If you'd like to use a TiDB version other than v5.0.2, cancel and retry with the following arguments:
+    Specify version manually:   tiup playground <version>
+    Specify version range:      tiup playground ^5
+    The nightly version:        tiup playground nightly
+
+Playground Bootstrapping...
+Start pd instance
+Start tikv instance
+Start tikv instance
+Start tikv instance
+PD client endpoints: [127.0.0.1:2379]
+To view the Prometheus: http://127.0.0.1:33703
+To view the Grafana: http://127.0.0.1:3000
+```
+
+## Step 2. Write data
+
+On another terminal session, we will use [go-ycsb](https://github.com/pingcap/go-ycsb) to launch a workload.
+
+1. Clone the `go-ycsb` from GitHub.
+    ```sh
+    git clone https://github.com/pingcap/go-ycsb.git
+    ```
+2. Build the application from the source.
+    ```sh
+    make
+    ```
+3. Load a small workload using `go-ycsb`
+    ```sh
+    # By default, this workload will insert 1000 records into TiKV
+    ./bin/go-ycsb load tikv -P workloads/workloada -p tikv.pd="127.0.0.1:2379" -p tikv.type="raw" 
+    ```
+
+## Step 3. Verify replication
+
+1. To understand replication in TiKV, it's important to review a few concepts from the [architecture](https://github.com/tikv/tikv#tikv-software-stack).
+
+| Concept     |                                                                                                           Description                                                                                                            |
+| ----------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| **Region**  | TiKV could seem like a giant sorted map of key-value pairs. The region is the basic unit of key-value data movement. Each region is a range of keys and replicated to multiple Nodes. These multiple replicas form a Raft group. |
+| **Replica** |                                                                  TiKV replicates each region (3 times by default) and stores each replica on a different node.                                                                   |
+
+2. With those concepts in mind, open the Grafana at [http://localhost:3000](http://localhost:3000) (printed from `tiup-playground`) and log in with `admin` whose password is the same as username.
+
+3. On the **playground-overview** dashboard, note the matrices in **region** panel in **TiKV** tab. It shows that the Regions count is the same on all three nodes. This indicates that:
+   *  There are this many "ranges" of data in the cluster. These are the small workload that we use `go-ycsb` injected.
```

Comment:
```suggestion
   *  There are this many `Region` in the cluster. These are the data imported by `go-ycsb`  which are then split by TiKV.
```

Reply:
There is no region split while we inject data to the cluster at the first time.

Reply:
Do you mean that after importing the data, there are only one region?

Reply:
Yes. It's a small workload that will create only one region.